src/main/kotlin/com/matthewrussell/trchunkbrowser/domain/ConvertDirectory.kt

```diff
@@ -8,29 +8,26 @@ class ConvertDirectory(private val root: File) {
         val outDir = outputDir.resolve("${root.name}-out")
         return Completable
             .fromAction {
-                // copy to the output directory
-                root.copyRecursively(outDir, overwrite = true)
-                convertInPlace(outDir)
+                convertInPlace(root, outDir)
             }
             .doOnError {
                 outDir.deleteRecursively()
             }
     }
 
-    private fun convertInPlace(dir: File) {
+    private fun convertInPlace(dir: File, out: File) {
         val contents = dir.listFiles().toList()
         val wavFiles = contents.filter {
             it.isFile && (it.extension == "wav" || it.extension == "WAV")
```

Comment:
I probably shouldn't comment on this since it's not part of the PR changes, but this could be it.extension.toLowerCase() == "wav"